website/app.js

```diff
@@ -0,0 +1,197 @@
+//const {platform} = require('os')
```

Comment:
Can this be removed?

Reply:
It's for [this bit](https://github.com/Farrser/AvoDocs/pull/60/files#diff-28743170234ea8cbb18d37078cbc5c3eR189) down below - so yeah could remove if needed

Reply:
Resolved in 010d57d32fc8ff250ce4a6a6e34908e8eb731e33